source/guide/3.2/reference-diamond-snmp-integration.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: bt_wiki
+title: Diamond SNMP integration
+category: Reference
+publish: true
+abstract: "Reference on how to use diamond monitoring with SNMP device"
+pageord: 1020
+---
+
+{%summary%}
+An extension to the diamond plugin that adds support for monitoring SNMP metrics on remote machines.
+The examples usage and necessary types are in [cloudify-diamond-snmp-extension](https://github.com/cloudify-cosmo/cloudify-diamond-snmp-extension)
+{%endsummary%}
+
+{%note title=Note%}
+Here you can find [Diamond plugin guide](plugin-diamond.html)
+{%endnote%}
+
+# SNMP types
+All node types you will need are defined in [snmp-types.yaml](https://github.com/cloudify-cosmo/cloudify-diamond-snmp-extension/blob/CFY-2305-snmp_diamond_integration/snmp-types.yaml). The security group necessary for OpenStack is defined in [openstack-snmp-types.yaml](https://github.com/cloudify-cosmo/cloudify-diamond-snmp-extension/blob/CFY-2305-snmp_diamond_integration/openstack-snmp-types.yaml). SNMP proxy is a node responsible for gathering the requested metrics from SNMP devices and sending them to RabbitMQ on behalf of those devices as if they were reporting those metrics by themselves (the proxy should be transparent).
```

Comment:
It should be specified that the [openstack-snmp-types.yaml](https://github.com/cloudify-cosmo/cloudify-diamond-snmp-extension/blob/CFY-2305-snmp_diamond_integration/openstack-snmp-types.yaml) is only provided as reference for usage in openstack. Also, the branch name in the link should at the very least be changed to master